README.md

```diff
@@ -1,19 +1,89 @@
 [![Pursuit Logo](https://avatars1.githubusercontent.com/u/5825944?s=200&v=4)](https://pursuit.org)
 
-# [Assignment Title]
+# Pursuit-Core-Web-Promises-Async-Await-Lab
 
 ## Getting Started
-* Fork this repo
-* Clone the forked repository
-* `cd` to the directory where you cloned it
-* `npm install` to install dependencies
-* `npm test` to open the cypress testing window
 
-> *Note*: Remember to `git add`, `git commit` and `git push` regularly
+- Fork this repo
+- Clone the forked repository
+- `cd` to the directory where you cloned it
+- `npm install` to install dependencies
+- `npm test` to open the cypress testing window
+
+> _Note_: Remember to `git add`, `git commit` and `git push` regularly
+
+## Lab: Five (or more, or less) Card Draw
+
+Today, we're going to be working with the [Deck of Cards API](https://deckofcardsapi.com/).
+It's a foundational skill for us, as programmers, to learn how any particular API works from some time testing it out, so take 15 minutes or so to look at the API documentation and play around with it in Postman.
+
+Today, we're going to create an app that does these things:
+
+- Creates and shuffles a new deck when the page loads.
+- Includes a button to draw some cards from that deck.
+- Displays those cards to the user.
+- When the user clicks the button again, draws another set of cards from that same deck.
```

Comment:
We will fix the instructions in house so no action required @JoshuaKGoldberg , just flagging that these are the key user stories to implement and we should highlight them much more. (And reduce the narrative and hints below.)